### DISCUSSION TITLE
Latest v1.7.5 source code for extension

### DISCUSSION THREAD

**Discussion by svankirk** [cline/cline#325] [Feature Requests]
Sorry if this is a newbie question, but Is there a way for me to get the latest source code used to build the extension?

**Comment by quinn** [comment]
The repo you created this discussion on is the source code? 

eg https://github.com/saoudrizwan/claude-dev/blob/main/src/ClaudeDev.ts

**Reply by svankirk** [reply]
That seems to be the case but it's now getting the right code so I don't know what's going on. I'm going to have to look at it more closely.  Thanks!
